### PR TITLE
added more basic customer fields

### PR DESCRIPTION
**Pull request by jared-fraser**
can be expanded to add addresses.

### CODE REVIEW

**Comment by jared-fraser** [comment]
I don't know why Go 1.7 failed on tests I didn't change...

File: customer.go [hunk 1]
```diff
@@ -2,6 +2,8 @@ package goshopify
 
 import (
 	"fmt"
+	"github.com/shopspring/decimal"
```
Comment (dlebech) [inline customer.go]: Minor: Could you separate this import from the standard library imports? Like [this](https://github.com/getconversio/go-shopify/blob/f33730dd1d730e846c20adaa93ac4b84fa315c8c/goshopify.go#L14-L16)

**Review by dlebech — COMMENTED** [review]
Thanks for the PR @jared-fraser . The reason the build fails is because the order tests use the `Customer` struct for the `customer` field in an order. It looks like e.g. `last_order_id`, `note` and `created_at` can be `null`. It looks like Golang 1.7 does not want to convert null time values.

I think you can probably fix the build by making `CreatedAt` and `UpdatedAt` pointers. Eventually, Golang 1.7 support will be dropped, but it would be nice to keep it for now.

**Comment by jared-fraser** [comment]
@dlebech I have updated the code. thanks for the feedback

**Review by stefanosala — APPROVED** [review]

**Review by dlebech — APPROVED** [review]

**Comment by dlebech** [comment]
Thanks @jared-fraser :100:

**Pull request merged**